docs/src/ci.md

```diff
@@ -0,0 +1,143 @@
+# Automatic Testing
+
+In the `QED.jl` eco-system, we use [Continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) (CIs) for running automatic tests. Each time, when a pull request is opened on GitHub.com or changes are committed to an existing pull request, the CI pipeline is triggered and starts a test script. The result of the tests will be reported in the pull request. In `QED.jl`, we distinguish between two kinds of tests
+
+- **Unit tests**: tests which check the functionality of the modified (sub) package. Those tests can either run standalone or use the functionality of other (third-party) packages.
```

Comment:
On line 109 in [ci/integTestGen/src/integTestGen.jl](https://github.com/QEDjl-project/QED.jl/pull/4/files#diff-f4034069be65108bf4437dfcddb7767e2eeabe52facb4a1172e2b661f74299c6) you wrote `(sub-)project`.
It should both be the same style, also line 57 here.

Reply:
You are right. I will use the term `(sub-) package`, if I talked about the dependencies  defined in a `project.toml`. If I talking about `(sub-) project`, it means the git repository.